src/extensionGlobals.ts

```diff
@@ -11,11 +11,21 @@ export namespace ext {
     /**
      * Commands which will be available when extension is activated
      */
-    export let askGeneralCommands: GenericCommand[];
+    export const askGeneralCommands: GenericCommand[] = [];
 
     /**
      * Commands which will be available when extension is activated
      * And when a skill is detected
      */
-    export let askSkillCommands: GenericCommand[];
+    export const askSkillCommands: GenericCommand[] = [];
+
+    /**
+     * Webviews which are registered in this extension
+     */
+    export const webViews: AbstractWebView[] = [];
+
+    /**
+     * TreeViews which are registered in this extension
+     */
+    export const treeViews: AbstractTreeView[] = [];
```

Comment:
This is good 👍 . We should also look into maintaining other components outside of the extension container (like event handlers, status bar items etc.) in this namespace.